api/Node.json

```diff
@@ -1699,7 +1699,7 @@
               }
             ],
             "ie": {
-              "version_added": "9"
+              "version_added": "8"
```

Comment:
It looks like this was actually supported much earlier than IE 8.  Using the [mdn-bcd-collector project](https://mdn-bcd-collector.appspot.com/tests/api/Node/ownerDocument) and BrowserStack, I was able to confirm support all the way back to IE 6, possibly further.  Can we set the version accordingly?

```suggestion
              "version_added": "≤6"
```

Reply:
It is safe after tested under my IE6

![image](https://user-images.githubusercontent.com/9573300/99204176-c9214580-27ef-11eb-8c1d-24cf487a7069.png)